tests/test_data_import.py

```diff
@@ -150,9 +148,9 @@ def test_calc_cycle_stats():
 
 def test_prepare_maccor_file():
     maccor_file = os.path.join(
-    rawTestDataPath, "cycle", "MACCOR_example", "MACCOR_example.txt")   
+    testDataBasePath, "cycle", "MACCOR_example", "MACCOR_example.txt")   
```

Comment:
Can you indent this line so that it's clear that it's a continuation from the previous line.